build/crds/build-crds.sh

```diff
@@ -45,9 +44,12 @@ copy_ob_obc_crds() {
   cp -f "${SCRIPT_ROOT}/cluster/olm/ceph/assemble/objectbucket.io_objectbuckets.yaml" "$OLM_CATALOG_DIR"
 }
 
-generating_crds() {
-  echo "Generating crds.yaml"
+generating_crds_v1() {
+  echo "Generating v1 in crds.yaml"
   "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="./pkg/apis/ceph.rook.io/v1" output:crd:artifacts:config="$OLM_CATALOG_DIR"
+}
+
+generating_crds_v1alpha2() {
   "$CONTROLLER_GEN_BIN_PATH" "$CRD_OPTIONS" paths="./pkg/apis/rook.io/v1alpha2" output:crd:artifacts:config="$OLM_CATALOG_DIR"
```

Comment:
Do we even need to generate the v1alpha2 anymore? If this just affects the csv, I don't think anyone is using the  flex driver in that case.

Reply:
This affects `crds.yaml` too.